### PR TITLE
Pop up

### PR DESCRIPTION
**Pull request by Mehria-Rahmani**
-While clicking  on(or taps) the button to check project details, the popup with details about the project appears.
-While clicking on (or taps) the close (X) button, the popup disappears.
-for both mobile and desktop screen sizes   popups is implemented.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,427 +1,275 @@
...
+        </p>
+        <ul class="usedTechs">
+          <li class="tech">CSS</li>
+          <li class="tech">HTML</li>
+          <li class="tech">Bootstrap</li>
+          <li class="tech">Ruby</li>
+        </ul>
+        <button class="mybtn recentbtn" id="recentbutton" type="button">
+          See Project
+        </button>
```
Comment (muneebulrehman) [inline index.html:141]: @Mehria-Rahmani , all the projects should be generated dynamically from Js. However, it seems your first project is typed in index.html and also doesn't have a popup. Kindly implement both the changes for the said section.
This is the card I am referring to. ⬇️
image

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @Mehria-Rahmani , 

Good job so far! I see you put a lot of effort into this project. However, there is this one issue that you still need to work on to go to the next project but you are almost there!

## Highlights
- Descriptive PR
- No linter errors
- Good code structure.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Mehria-Rahmani) [reply index.html:141]: -hello, as we see in project requirements there is no need to make popup for this one , kindly re review it.

**Comment by Mehria-Rahmani** [comment]
Hello
As we see in project requirements there is no need to make pop up for this
sectio  , kindly re review it,  and as you know we dont have much time to
do everything i just want to complete the requirements,  and i will work on
it in my free time, thank you

File: jsfiles/card.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
+const projects = {
+  first: {
+    name: 'First Project',
+    desc: `first project description <br> Lorem Ipsum is simply. 
+           Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+           when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.`,
```
Comment (koachcode) [inline jsfiles/card.js:6]: - [Optional] Please add the missing popup as the previous reviewer requested, so as to make your portfolio looks professional and presentable. Happy coding :+1:

**Review by koachcode — APPROVED** [review]
Hi @Mehria-Rahmani , 

# STATUS: APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Mehria-Rahmani** [comment]
Thank you so much

**Pull request merged**